cpp/velox/shuffle/VeloxShuffleWriter.cc

```diff
@@ -1097,7 +1101,7 @@ arrow::Status VeloxShuffleWriter::splitFixedWidthValueBuffer(const velox::RowVec
           binaryBuf.valueCapacity = capacity;
           dstValuePtr = binaryBuf.valuePtr + valueOffset - stringLen;
 
-          std::cout << "Split value buffer resized colIdx" << binaryIdx << std::endl;
+          DLOG(INFO) << "Split value buffer resized. Column index: " << binaryColumnIndices_[binaryIdx] << std::endl;
```

Comment:
@FelixYBW Our CI now prints too many logs from this line. Can we remove the log here?

cc: @Yohahaha https://github.com/oap-project/gluten/issues/3208#issuecomment-1726854275

Reply:
Let's remove the log here. `VsPrintSplit` at below has given enough message.